packages/blast-core/README.md

```diff
@@ -337,6 +338,69 @@ async function main () {
   // ...
 ```
 
+---
+## Creating a custom task
```

Comment:
Do you think it is a good idea to include custom tasks in the "Help and version" section's description? Something like "...to show all available subcommands, custom tasks, parameters..."

Reply:
I think that 'Run --help or help on any blast command to show all available subcommands, parameters and additional information.' says it all